QuickGuidePanels/useToggleRole.js

```diff
@@ -0,0 +1,23 @@
+import {useRef, useCallback} from 'react';
```

Comment:
Instead of creating a new file, Please move it to the internals/Panels/ folder.

Reply:
I moved files to internal/Panels.